addons/l10n_latam_invoice_document/models/account_move.py

```diff
@@ -193,7 +193,7 @@ def _compute_l10n_latam_available_document_types(self):
     @api.depends_context('internal_type')
     def _compute_l10n_latam_document_type(self):
         internal_type = self._context.get('internal_type', False)
-        for rec in self.filtered(lambda x: x.state == 'draft'):
+        for rec in self.filtered(lambda x: x.state == 'draft' and not x.l10n_latam_document_number):
```

Comment:
This check doesn't guarantee that the invoice was posted before, as `l10n_latam_document_number` can be set on draft invoices

Reply:
@alialfie i think in Argentinian localization l10n_latam_document_number is set only on posted invoices.

Reply:
Hmm, not sure about this 
![image](https://github.com/user-attachments/assets/33e95c0a-da43-468b-9c0a-15b6ab476586)
As you can see, on runbot I can create an invoice in the AR localisation and the `l10n_latam_document_number` can be set even if it's in draft. Also, the changes in the code will affect not just the AR localisation, but any module that uses the `l10n_latam_invoice_document`.